package.json

```diff
@@ -94,5 +94,8 @@
     "sinon": "^1.17.4",
     "sinon-as-promised": "^4.0.0",
     "sinon-chai": "^2.8.0"
+  },
+  "optionalDependencies": {
```

Comment:
https://docs.npmjs.com/files/package.json#optionaldependencies. Essentially, the whole installation of firebase-tools won't fail if this package fails to be installed for whatever reason.